### PR TITLE
Change TEvLocal actorsystem metrics

### PR DESCRIPTION
**Pull request by kruall**
### Changelog entry 

...

### Changelog category 

* Bugfix 

### Additional information

...

### CODE REVIEW

**Review by alexvru — DISMISSED** [review]

File: ydb/core/protos/local.proto [hunk 1]
```diff
@@ -53,6 +63,7 @@ message TEvStatus {
     optional uint64 AvailableWeight = 5;
     optional NKikimrTabletBase.TMetrics ResourceMaximum = 8;
     optional uint64 StartTime = 7;
+    optional TActorSystemInfo ActorSystemInfo = 9;
```
Comment (vporyadke) [inline ydb/core/protos/local.proto:66]: TEvStatus is sent when Local first registers in Hive, and when the tenant is altered. If we want this info to be sent regularly, I would put in TabletMetrics message.

File: ydb/core/mind/local.cpp [hunk 2]
```diff
@@ -111,6 +111,9 @@ class TLocalNodeRegistrar : public TActorBootstrapped<TLocalNodeRegistrar> {
     ui64 MemLimit = 0;
     double NodeUsage = 0;
 
+    TInstant LastUpdate;
```
Comment (vporyadke) [inline ydb/core/mind/local.cpp:114]: The name is confusing, last update of what? Also If we change it to use TabletMetrics message, there already is a SendTabletMetricsTime member

File: ydb/core/mind/local.cpp [hunk 3]
```diff
@@ -278,15 +281,35 @@ class TLocalNodeRegistrar : public TActorBootstrapped<TLocalNodeRegistrar> {
...
+        ctx.ExecutorThread.ActorSystem->GetExecutorPoolStates(poolStates);
+
+        TDuration passedTime = ctx.Now() - LastUpdate;
+        LastUpdate = ctx.Now();
+
+        auto *actorSystemInfo = record.MutableActorSystemInfo();
+        double cores = 0;
+        for (ui8 poolId = 0; poolId < poolStates.size(); ++poolId) {
+            auto &poolState = poolStates[poolId];
+            if (poolId != AppData()->IOPoolId) {
```
Comment (vporyadke) [inline ydb/core/mind/local.cpp:294]: We already have a setting PoolsToMonitorForUsage. I believe it should be used here. But it also excludes Batch pool by default - is there a reason you're including it here?

**Review by vporyadke — CHANGES_REQUESTED** [review]

File: ydb/core/protos/local.proto [hunk 4]
```diff
@@ -32,6 +32,16 @@ message TLocalConfig {
     repeated NKikimrSchemeOp.TResourceProfile ResourceProfiles = 1;
 }
 
+message TActorSystemInfo {
```
Comment (vporyadke) [inline ydb/core/protos/local.proto:35]: Currently we send a single number TotalNodeUsage, that is the maximum over pool usages (and mem usage).
What do we expect to gain from sending this detailed info? Just seems like we will simply do the calculate usage + take maximum logic on Hive side instead.

**Review by vporyadke — APPROVED** [review]

**Review by alexvru — APPROVED** [review]

**Pull request merged**